ModernWallet/Classes/ViewController/CashOut/CashOurBankAccountDetailsViewController.swift

```diff
@@ -37,6 +52,10 @@ class CashOurBankAccountDetailsViewController: UIViewController {
         bicTextField.placeholder = Localize("cashOut.newDesign.bic")
         accountHolderTextField.placeholder = Localize("cashOut.newDesign.accountHolder")
         currencyTextField.placeholder = Localize("cashOut.newDesign.accountHolderAddress")
+        accountHolderCountryTextField.placeholder = Localize("cashOut.newDesign.accountHolderCountry")
```

Comment:
Please move all localisations in a separate function in ViewController Extension